### PR TITLE
[FIX] Advanced trade: fix values calculations

### PR DESCRIPTION
**Pull request by darkdarkdragon**
Do not convert amount and order values to number
when assigning to input - because chrome and IE shows
big/small numbers in input in scientific notation

use "replace(/,/g, '')" on result of "to_text_full()"
until ripple lib fixed - "to_text_full()" returns
string with group separator

### CODE REVIEW

**Comment by vhpoet** [comment]
"until ripple lib fixed - "to_text_full()" returns"
@geertweening @clark800 do you guys know what needs to be fixed?

**Comment by clark800** [comment]
He's referring to this pull request: https://github.com/ripple/ripple-lib/pull/326

**Comment by darkdarkdragon** [comment]
@vhpoet just come to my mind - why to_text_full was used here in the first place? 
Anyway result is going to from_json, so why not just use from_json?

File: src/js/tabs/trade.controller.js [hunk 1]
```diff
@@ -834,7 +836,7 @@ TradeTab.prototype.angular = function(module)
       if (order.price_amount && order.price_amount.is_valid() &&
           order.first_amount && order.first_amount.is_valid()) {
         order.second_amount = order.price_amount.product_human(order.first_amount);
-        order.second = +order.second_amount.to_human({group_sep: false});
+        order.second = order.second_amount.to_human({group_sep: false});
```
Comment (jublo) [inline src/js/tabs/trade.controller.js]: Why was the `+` killed? It's a conversion to number.

File: src/js/tabs/trade.controller.js [hunk 2]
```diff
@@ -852,15 +854,16 @@ TradeTab.prototype.angular = function(module)
...
 
-          order.first_amount = Amount.from_json(order.second_amount.to_text_full()).ratio_human(Amount.from_json(price + '/' + currency + '/' + issuer), {reference_date: new Date()});
+          // use replace(/,/g,'') until ripple lib fixed
+          order.first_amount = Amount.from_json(order.second_amount.to_text_full().replace(/,/g, '')).ratio_human(Amount.from_json(price + '/' + currency + '/' + issuer), {reference_date: new Date()});
         } else {
-          order.first_amount = Amount.from_json(order.second_amount.to_text_full()).ratio_human(Amount.from_json(order.price_amount.to_text()), {reference_date: new Date()});
+          order.first_amount = Amount.from_json(order.second_amount.to_text_full().replace(/,/g, '')).ratio_human(Amount.from_json(order.price_amount.to_text()), {reference_date: new Date()});
         }
-        order.first = +order.first_amount.to_human({group_sep: false});
+        order.first = order.first_amount.to_human({group_sep: false});
```
Comment (jublo) [inline src/js/tabs/trade.controller.js]: See above.

File: src/js/tabs/trade.controller.js (hunk 1 above)
Comment (darkdarkdragon) [inline src/js/tabs/trade.controller.js]: @jublo That was whole point of this commit. This value goes into input field. And when it is number, chrome shows it in e-notation if it too small or too big

Comment (jublo) [inline src/js/tabs/trade.controller.js]: Understood. Well done then.

**Comment by vhpoet** [comment]
LGTM

**Comment by mrajvanshy** [comment]
LGTM

**Pull request merged**